src/views/conference/2018/index/l10n.json

```diff
@@ -11,7 +11,7 @@
 
     "conference-2018.desc1": "Join us for the Scratch@MIT conference, a playful gathering of educators, researchers, developers, and other members of the worldwide Scratch community.",
     "conference-2018.desc2": "We're planning a very participatory conference, with an entire day of hands-on workshops and lots of opportunities for peer-to-peer discussion and collaboration. The conference is intended primarily for adults who support young people learning Scratch.",
-    "conference-2018.registrationTime": "Registration opens March 1, 2018.",
+    "conference-2018.registrationdate": "Registration opens March 1, 2018.",
```

Comment:
This should be a capital `d`: `conference-2018.registrationDate` (not `conference-2018.registrationdate`).

Reply:
@redmangospros